chore: add doc for creating new scan (#429)
cds-snc/scan-websites: README.md

   - Build and push container to ECR (Push to Production)
- Profit :tada:

## Adding new scan

- https://scan-websites.alpha.canada.ca/en/template > New
- Create the new scan with the selected scanner
- Grab the token for the template scan created

> - Create a step in the project you would like to scan

Reviewer: ```suggestion
- Create a scheduled workflow in the project you would like to scan:
```yml
name: Scan for security vulnerabilities (Weekly on Sunday)

on:
  schedule:
  - cron: "0 12 * * 0"

jobs:
  security-scan:
    name: Run scan websites (security)
    uses: cds-snc/scan-websites/.github/workflows/start_scan.yml@main
    with:
      dynamic: true
    secrets:
      scan_websites_key: ${{ secrets.SCAN_WEBSITES_KEY }}
      scan_websites_template: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}
```

> -  `curl -X GET -H 'X-API-KEY: ${{ secrets.SCAN_WEBSITES_KEY }}' -H 'X-TEMPLATE-TOKEN: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}' https://scan-websites.alpha.canada.ca/scans/start`

Reviewer: ```suggestion
-  You can also trigger a scan directly in a workflow with:
```sh
curl -X GET -H 'X-API-KEY: ${{ secrets.SCAN_WEBSITES_KEY }}' -H 'X-TEMPLATE-TOKEN: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}' https://scan-websites.alpha.canada.ca/scans/start`
```
```

   - Build and push container to ECR (Push to Production)
- Profit :tada:

## Adding new scan

- https://scan-websites.alpha.canada.ca/en/template > New
- Create the new scan with the selected scanner
- Grab the token for the template scan created
- Create a scheduled workflow in the project you would like to scan:
```yml
name: Scan for security vulnerabilities (Weekly on Sunday)

on:
  schedule:
  - cron: "0 12 * * 0"

jobs:
  security-scan:
    name: Run scan websites (security)
    uses: cds-snc/scan-websites/.github/workflows/start_scan.yml@main
    with:
      dynamic: true
    secrets:
      scan_websites_key: ${{ secrets.SCAN_WEBSITES_KEY }}
      scan_websites_template: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}
-  You can also trigger a scan directly in a workflow with:
```sh

> curl -X GET -H 'X-API-KEY: ${{ secrets.SCAN_WEBSITES_KEY }}' -H 'X-TEMPLATE-TOKEN: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}' https://scan-websites.alpha.canada.ca/scans/start`

Reviewer: ```suggestion
curl -X GET -H 'X-API-KEY: ${{ secrets.SCAN_WEBSITES_KEY }}' -H 'X-TEMPLATE-TOKEN: ${{ secrets.SCAN_WEBSITES_TEMPLATE }}' https://scan-websites.alpha.canada.ca/scans/start
```
```
